### DISCUSSION TITLE
Validate file stream contents with content type

### DISCUSSION THREAD

**Discussion by panchmp** [directus/directus#8125] [Feature Requests]
When sending a wrong/illegal file content string for a multipart form-data file blob, the API will just save the file and return the created directus_files record.

---

The wrong request returned random FileObject instead of 400 Bad Request

### To Reproduce

```
curl  
  -X POST "https://directus.somesite.com/files"
  -H "Authorization: Bearer XXXXXXXXXXXXXXXXX"
  -H "Accept: application/json"
  -H "Content-Type: multipart/form-data" 
  -F "file=somefile.jpg;type=image/jpg"
```

return random FileObject instead of 400 Bad Request

I miss @ for -F "file=somefile.jpg;type=image/jpg" so the value is just string instead of a binary stream)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
